### PR TITLE
Adds KernelRegistry Test

### PR DESCRIPTION
**Pull request by fiiiu**

### CODE REVIEW

File: kernel/test/KernelRegistry.test.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    await this.kernelRegistry.addInstance(this.kernelInstance.address);
+    const instance = await this.kernelRegistry.getInstance(name, version_0);
+    assert.equal(instance, this.kernelInstance.address);
+  });
+
+  it('should only allow owner to add instances', async function () {
+    await assertRevert(this.kernelRegistry.addInstance(this.kernelInstance.address, { from: other }));
+  });
+
+  it('should not allow repeated instances', async function () {
```
Comment (spalladino) [inline kernel/test/KernelRegistry.test.js]: Let's add a test for adding a different instance but with the same name/version

File: kernel/test/KernelRegistry.test.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+contract('KernelRegistry', ([owner, other]) => {
+  const name = "Test";
+  const version_0 = "0.0";
+  const version_1 = "0.1";
+
+  beforeEach(async function () {
+    this.kernelInstance = await KernelInstance.new(name, version_0, 0);
+    await this.kernelInstance.freeze();
+    this.kernelInstance2 = await KernelInstance.new(name, version_1, this.kernelInstance.address);
```
Comment (facuspagnuolo) [inline kernel/test/KernelRegistry.test.js]: what about using `kernelInstance` and `anotherKernelInstance` or `kernelInstance_0` and `kernelInstance_1`?

File: kernel/test/KernelRegistry.test.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    this.kernelRegistry = await KernelRegistry.new();
+  });
+
+  it('should add single instance', async function () {
+    await this.kernelRegistry.addInstance(this.kernelInstance.address).should.be.fulfilled;
```
Comment (facuspagnuolo) [inline kernel/test/KernelRegistry.test.js]: shouldn't we check that `KernelRegistry#getInstance` works as expected after adding a new instance?

Comment (fiiiu) [reply kernel/test/KernelRegistry.test.js]: mm not sure what you mean, isn't this tested in `'should correctly recover instances'`? Or are you saying after adding _two_ instances?

**Pull request merged**